### ISSUE TITLE
Output logits instead of probs for NLL

### ISSUE THREAD

**Issue by nanbhas** [StanfordMIMI/RaLEs#1]
https://github.com/JMZAM/RaLEs/blob/main/evaluation/negloglikelihood.py

- [x] Add support for both probs and logits
- [ ] Update _info for logits
